docs/ml-classification-regression.md

```diff
@@ -27,10 +27,10 @@ displayTitle: Classification and regression in spark.ml
 * This will become a table of contents (this text will be scraped).
 {:toc}
 
-In MLlib, we implement popular linear methods such as logistic
+In `spark.ml`, we implement popular linear methods such as logistic
```

Comment:
Not this file; the "ml-" prefix ones are for spark.ml.  (It's true the functionality is almost the same currently, but it's a bit different and will diverge more.)